metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1416,6 +1424,29 @@ class MetalsLanguageServer(
         scribe.debug(s"Executing SuperMethodHierarchy ${command}")
         supermethods.jumpToSelectedSuperMethod(params).asJavaObject
 
+      case ServerCommands.ResetChoicePopup() =>
+        val argsMaybe = Option(params.getArguments())
+        (argsMaybe
+          .map { args =>
+            args.asScala.headOption match {
+              case Some(argObject: JsonPrimitive) =>
+                val arg = argObject.asInstanceOf[JsonPrimitive]
+                val value = arg.getAsString()
+                scribe.info(
```

Comment:
```suggestion
                scribe.debug(
```
no need to always write it.

Reply:
:+1: